services/client/options.go

```diff
@@ -30,16 +30,14 @@ import (
 )
 
 const (
-	defaultHeartbeatCheckInterval = 10 * time.Second
-	defaultInitTimeout            = 5 * time.Second
-	defaultGaugeInterval          = 10 * time.Second
+	defaultInitTimeout   = 15 * time.Second
```

Comment:
because the Get() in kv store first calls etcd with a default timeout of 10 seconds, after the etcd call is timed out the kv store will fall back to the cache, so the initTimeout should be larger than 10 seconds here to really leverage the cache. I'll add a comment for it